source/Orchestrator/check_ssm_execution.py

```diff
@@ -112,6 +112,45 @@ def valid_automation_doc(automation_doc):
        "ControlId" in automation_doc and \
        "AccountId" in automation_doc
 
+def get_execution_log(response_data):
+    logdata = []
+    if 'ExecutionLog' in response_data:
+        logdata = response_data['ExecutionLog'].split('\n')
+
+    return logdata
+
+def get_affected_object(response_data):
+    affected_object_out = 'UNKNOWN'
+    if "ParseInput.AffectedObject" in response_data:
+        affected_object = response_data.get('ParseInput.AffectedObject')[0]
+        try:
+            affected_object = json.loads(affected_object)
+            if 'Type' in affected_object and 'Id' in affected_object:
+                affected_object_out = affected_object['Type'] + ' ' + affected_object['Id']
+            else:
+                affected_object_out = str(affected_object)
+        except JSONDecodeError:
+            print('Expected serialized json, got ' + str(affected_object))
+            affected_object_out = str(affected_object)
+
```

Comment:
recommendation for next PR to further reduce complexity:

I would cut the affect_object_out variable and instead repalce it with explicit returns using a fail-fast gate (or whatever it's called). You can do this by replacing
```
affected_object_out = 'UNKNOWN'
if "ParseInput.AffectedObject" in response_data:
  //mutate affected_object_out before returning
```

with
```
if "ParseInput.AffectedObject" not in response_data:
  return 'UNKNOWN'

//rest of the original function here but replace any mutations of affected_object_out with explicit returns
```

Reply:
In general your recommendation is a better way to write code the first time around. But keep in mind the scope and context of this change. I would much rather see something moved around and not changed in the first PR, than moved and changed in the same PR.